whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapExceptionMapper.java

```diff
@@ -38,7 +33,6 @@ public RdapExceptionMapper(
 
     @Override
     public Response toResponse(final Exception exception) {
-
         if (exception instanceof IllegalSourceException) {
             return Response.status(HttpServletResponse.SC_BAD_REQUEST).entity(createErrorEntity(HttpServletResponse.SC_BAD_REQUEST, exception.getMessage())).build();
```

Comment:
can you call createErrorResponse directly?

Reply:
IllegalSourceException extends IllegalArgumentException so I deleted it as well